morebits.js

```diff
@@ -3848,6 +3850,7 @@ Morebits.wiki.page = function(pageName, status) {
 		var rev = response.pages[0].revisions && response.pages[0].revisions[0];
```

Comment:
Shouldn't we provide the failure function to `fnCheckPageName` on line 3846?

Reply:
things get hazy when there are that many lines ....